test/associations_test.rb

```diff
@@ -98,5 +114,22 @@ def test_no_duplicated_associations
         .associations
         .must_equal expected
     end
+
+    def test_includes_over_default_associations
+      peggy = Person.new
+      roger = Person.new(id: 2, name: 'Roger', occupation: 'Chief')
+      daniel = Person.new(id: 3, name: 'Daniel', occupation: 'Agent')
+      peggy.colleagues = [daniel]
+      daniel.superior = roger
+      expected = {
+        colleagues: Collection.new([daniel], root: false, includes: :superior)
+          .as_json,
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.